src/views/Home/SystemIntakeBanners.tsx

```diff
@@ -77,6 +77,14 @@ const SystemIntakeBanners = () => {
     NO_GOVERNANCE: {
       title: t('banner.title.responseRecevied'),
       description: t('banner.description.checkNextStep')
+    },
```

Comment:
This will likely change with future work from Chris, but wanted to make sure we displayed something in the meantime.